contracts/BosonRouter.sol

```diff
@@ -8,10 +8,12 @@ import "@openzeppelin/contracts/utils/Pausable.sol";
 import "@openzeppelin/contracts/utils/Address.sol";
 import "./interfaces/IVoucherKernel.sol";
 import "./interfaces/IERC20WithPermit.sol";
-import "./interfaces/IFundLimitsOracle.sol";
+import "./interfaces/ITokenRegistry.sol";
 import "./interfaces/IBosonRouter.sol";
 import "./interfaces/ICashier.sol";
+import "./interfaces/ITokenWrapper.sol";
 import "./UsingHelpers.sol";
+import "hardhat/console.sol";
```

Comment:
`console` could be removed

Reply:
Removed. Will be included in next commit.